handlers/bot_handler.py

```diff
@@ -54,14 +54,36 @@ def execute(cls, slack_wrapper, args, timestamp, channel_id, user_id, user_is_ad
             raise InvalidCommand("Sorry, couldn't retrieve the git information for the bot...")
 
 
+class InviteCommand(Command):
+    """
+    Invite a list of members to the current channel, parses out members already
+    present
+    """
+
+    @classmethod
+    def execute(cls, slack_wrapper, args, timestamp, channel_id, user_id, user_is_admin):
+        """Execute the Invite command"""
```

Comment:
get rid of this, adds nothing

Reply:
the other methods have a similar docstring, just fyi

Reply:
Yep, but we should move towards correcting. You can correct them all if you like. See [here](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings) for the convention we try to follow.

Reply:
Oh yeah we should get rid of this, kileak likes adding docstrings for everything, I am trying to clean them up :) look at my review of his current PR.

Reply:
tbh, i liked them when i was learning the codebase. they might seem a little redundant, but reading docstrings is friendlier than making assumptions about the conditions that call certain methods

Reply:
At the very least at a period for pep257